tests/system/data_sources/test_oracle.py

```diff
@@ -669,11 +716,10 @@ def test_custom_query_invalid_long_decimal():
     new=mock_get_connection_config,
 )
 def test_custom_query_row_validation_oracle_to_postgres():
-    # TODO Change hash_cols below to include col_tstz when issue-706 is complete.
-    # TODO col_raw/col_long_raw are blocked by issue-773 (is it even reasonable to expect binary columns to work here?)
     # TODO Change hash_cols below to include col_nvarchar_30,col_nchar_2 when issue-772 is complete.
     # TODO Change hash_cols below to include col_interval_ds when issue-1214 is complete.
-    # TODO Change hash_cols below to include col_clob/col_nclob/col_blob/col_json/col_jsonb when issue-1364 is complete.
+    # TODO Change hash_cols below to include col_clob/col_nclob/col_blob when issue-1364 is complete.
+    # TODO Change hash_cols below to include col_json/col_jsonb when issue-1338 is complete.
     # Excluded col_float32,col_float64 due to the lossy nature of BINARY_FLOAT/DOUBLE.
     hash_cols = ",".join(
```

Comment:
```suggestion
    # Excluded col_long_raw because LONG types are not supported.
    hash_cols = ",".join(
```
Adding here same comment that was added in some methods above